typescript/sdk/src/providers/MultiProvider.ts

```diff
@@ -29,16 +30,22 @@ const DEFAULT_RETRY_OPTIONS: RetryProviderOptions = {
   baseRetryMs: 250,
 };
 
+const DEFAULT_PROVIDER_TIMEOUT: number = 1000 * 60 * 5; // 5 min
```

Comment:
Two things here:
1. Please export this from `src/index.ts`
2. 5 minutes seems very high. It's 5min for a single request? I expect if something doesn't return in 30s, it won't every succeed

Reply:
My understanding is it is the timeout for waiting for a txn. See the linked code in the description.

Reply:
I think your link in the description goes to Ethers v6, which is pretty new. We're using V5.

The timeout property starts [here in JSONRpcProvider](https://github.com/ethers-io/ethers.js/blob/v5.7/packages/providers/src.ts/json-rpc-provider.ts#L412), then down to [fetchJson](https://github.com/ethers-io/ethers.js/blob/v5.7/packages/web/src.ts/index.ts#L362), then [fetchData](https://github.com/ethers-io/ethers.js/blob/v5.7/packages/web/src.ts/index.ts#L91), where it uses a [default of 2 minutes](https://github.com/ethers-io/ethers.js/blob/v5.7/packages/web/src.ts/index.ts#L116) if no override is set. 

Is the intention to increase or decrease that default here?

Reply:
I guess I was looking at the wrong code, these changes are not needed then.